charts/backingservices/values.yaml

```diff
@@ -44,14 +44,14 @@ srs:
     #port: 9200
     #protocol: https
     # The elasticsearch connection supports three authentication methods: basic authentication , AWS IAM role-based authentication and Elasticsearch secure connection(tls).
-    # To configure tls on internal cluster uncomment and set `srs.srsStorage.tls.enabled: true`
+    # set `srs.srsStorage.tls.enabled: true` to enable tls on internal/external elasticsearch cluster.
     tls:
      enabled: false
     # Specify the certificate name used to connect to elasticsearch when tls.enabled: true and srsStorage.provisionInternalESCluster: false.
```

Comment:
To specify a certificate used to authenticate an external Elasticsearch service (with `tls.enabled: true` and `srsStorage.provisionInternalESCluster: false`), uncomment the following line to specify the TLS certificate name for your Elasticsearch service.